src/components/Article.jsx

```diff
@@ -1 +1,29 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = (props) => {
```

Comment:
```suggestion
export const Article = ({ article }) => {
```

Reply:
and then use destructuring to get keys you need
`const { title, date, text } = article`